lib/ddtrace/tracer.rb

```diff
@@ -406,7 +406,9 @@ def activate_priority_sampling!(base_sampler = nil)
                  else
                    PrioritySampler.new(
                      base_sampler: base_sampler,
-                     post_sampler: Datadog::RateByServiceSampler.new(1.0, env: proc { tags[:env] })
+                     post_sampler: Sampling::RuleSampler.new(
+                       default_sampler: RateByServiceSampler.new(1.0, env: -> { tags[:env] })
```

Comment:
do we need to configure this here if the default is `RateByServiceSampler.new(1.0, env: -> { Datadog.tracer.tags[:env] })` in `RuleSampler`?

Reply:
We don't really need to, I tried without it at first, but I had issues with circular references on the `Datadog.tracer.tags[:env]` part.

I'm trying something else now, we'll see if it works.

Reply:
ah, ok